doc/guide.rst

```diff
@@ -750,3 +750,53 @@ an input to the above mentioned genotyping commands. This can be done by running
     whatshap find_snv_candidates ref.fasta input.bam -o variants.vcf
 
 If Nanopore reads are used for calling SNPs, it is recommended to add option --nanopore to the above command.
+
+Polyploid Phasing
+=================
+
+In addition to diploid phasing, WhatsHap also supports polyploid phasing 
+through a different algorithm. The ``whatshap polyphase`` command works 
+almost the same as the ``phase`` command with a few restrictions:
+
+1. An additional integer argument ``--ploidy`` must be specified. This ploidy
+must match the ploidy in the provided VCF file(s). The ploidy also greatly
+impacts the running time as the phasing becomes more complex. Ploidies
+higher than 6 may take very long to process.
```

Comment:
Can we be more specific here and possibly mention example runtimes?

Reply:
We only have example runtimes for our multi-human sample. Could be a bit confusing to state them here, because this is not what you would usually phase with polyphase.